message_definitions/v1.0/common.xml

```diff
@@ -2218,6 +2218,67 @@
         <param index="6">Empty.</param>
         <param index="7">Empty.</param>
       </entry>
+      <!-- External AHRS commands (32000 to 32099) -->
+      <entry value="33000" name="MAV_CMD_EAHRS_START" hasLocation="false" isDestination="false">
+        <description>Send command "start data sending" to external AHRS</description>
+        <param index="1">Empty.</param>
+        <param index="2">Empty.</param>
+        <param index="3">Empty.</param>
+        <param index="4">Empty.</param>
+        <param index="5">Empty.</param>
+        <param index="6">Empty.</param>
+        <param index="7">Empty.</param>
+      </entry>
+      <entry value="33001" name="MAV_CMD_EAHRS_STOP" hasLocation="false" isDestination="false">
+        <description>Send command "stop data sending" to external AHRS</description>
+        <param index="1">Empty.</param>
+        <param index="2">Empty.</param>
+        <param index="3">Empty.</param>
+        <param index="4">Empty.</param>
+        <param index="5">Empty.</param>
+        <param index="6">Empty.</param>
+        <param index="7">Empty.</param>
+      </entry>
+      <entry value="33002" name="MAV_CMD_EAHRS_ENABLE_GNSS" hasLocation="false" isDestination="false">
+        <description>Send command "enable GNSS" to external AHRS</description>
+        <param index="1">Empty.</param>
+        <param index="2">Empty.</param>
+        <param index="3">Empty.</param>
+        <param index="4">Empty.</param>
+        <param index="5">Empty.</param>
+        <param index="6">Empty.</param>
+        <param index="7">Empty.</param>
+      </entry>
+      <entry value="33003" name="MAV_CMD_EAHRS_DISABLE_GNSS" hasLocation="false" isDestination="false">
+        <description>Send command "disable GNSS" to external AHRS</description>
+        <param index="1">Empty.</param>
+        <param index="2">Empty.</param>
+        <param index="3">Empty.</param>
+        <param index="4">Empty.</param>
+        <param index="5">Empty.</param>
+        <param index="6">Empty.</param>
+        <param index="7">Empty.</param>
+      </entry>
+      <entry value="33004" name="MAV_CMD_EAHRS_START_VG3D_CLB_IN_FLIGHT" hasLocation="false" isDestination="false">
```

Comment:
What's VG3D?

Reply:
Sorry, it's internal naming for magnetometer calibration

It's Ok, if I name this
MAV_CMD_EAHRS_START_MAG_CLB_IN_FLIGHT
MAV_CMD_EAHRS_STOPT_MAG_CLB_IN_FLIGHT
?

Reply:
Yep, if it's generic that's preferred. From what I know, we avoid vendor specific packets or vendor specific acronyms in the common.xml

Reply:
I removed these magnetometer calibration commands from this PR

If this PR will be merged I plan to open the new one with "magnetometer calibration" commands/handler